src/crash.asm

```diff
@@ -44,22 +44,31 @@ org $808573
     JML CrashHandler
 
 ; Hijack native COP vector
-org $00FFE4
-    dw COPHandler
-
-; Hijack emulation COP vector
-org $00FFF4
+org $80FFE4
+NativeCOPHandlerHook:
     dw COPHandler
 
 ; Hijack native BRK vector
-org $00FFE6
+org $80FFE6
+NativeBRKHandlerHook:
     dw BRKHandler
 
+; Hijack emulation COP vector
+org $80FFF4
+EmulationCOPHandlerHook:
+    dw COPHandler
+
 ; Hijack emulation IRQ/BRK vector
-; Don't do this. A bug in vanilla code causes this to make Mini-Kraid cry
+; Due to a bug in vanilla code, MSB needs to be unchanged (0x85)
 ; https://patrickjohnston.org/bank/A6?just=9BB2
-;org $00FFFE
-;    dw BRKHandler
+org $80FFFE
```

Comment:
The bug still occurs with this change since bank $00 is a mirror of $80.

Reply:
It does?  I was thinking the AND $00FFFF would pick up the $85 out of $8577, same as it would if we left it at the vanilla $8573.

Reply:
Oh, I see what you did below it now. Unfortunately, I can't test it again atm.